kubernetes/bluemix/single-tenant/deploy_vgw_v102x.json

```diff
@@ -0,0 +1,175 @@
+{
```

Comment:
Can this name be renamed to be `kubernetes/bluemix/single-tenant/deploy.json`, this way its the default file users use when using the latest images.

Reply:
so you would prefer this to be the default deploy.json and have the previous version of the file renamed deploy_vgw_v1.0.x,json?

Reply:
it can be done of course if that's your preference

Reply:
yup - it should be the new default deploy script

Reply:
I have updated the multinode deployment scripts as well and I renamed the files